src/info.c

```diff
@@ -18,14 +18,19 @@
  */
 
 #define _GNU_SOURCE
-#include <stdio.h>
-
 #include "swupd.h"
+#include <stdio.h>
```

Comment:
Usually it's better to include local headers after includes from the library. Is there a reason you changed that?

Reply:
And actually it looks like you don't even need that include

Reply:
Looks it was already there in master